README.md

```diff
@@ -165,23 +168,77 @@ In Device Simulator Express, you can use keyboard to interact with the device:
   - Acceleration detection
   - Temperature sensor
 
+### Useful Links
+- Tutorials and Example Code for BBC micro:bit:
+  - [MicroPython documentation](https://microbit-micropython.readthedocs.io/en/latest/)
+  - [BBC micro:bit examples on the official micro:bit website](https://microbit.org/projects/make-it-code-it/?filters=python)
+
 ### How to use
+
+#### 1. Start with the "Device Simulator Express [micro:bit]: New File" Command.
+
+1. Type in `"Device Simulator Express: [micro:bit] New File"` in the command palette (`CTRL/CMD + SHIFT + P` to open the command palette).  
+   <img alt='"New File" animation' src='assets/readmeFiles/microbit/microbit-new-file.gif'>
+2. Name and save your file somewhere, and we’re good to go!
+
+#### 2. Start from an existing python file.
+
+1. Open the folder or your .py file in Visual Studio Code.
+2. Run `Device Simulator Express: [micro:bit] Open Simulator` from the command palette or icon in the editor toolbar.
+
+#### 3. Run your code on the simulator.
+
+<img alt='How to run the simulator animation' src='assets/readmeFiles/microbit/microbit-run.gif'>

```

Comment:
_&ast;nit&ast;_
This gif and the cpx gif for step 3 don't have the red circle show up on click.

Reply:
Those red clicks are from `screencast mode` and don't work on the webview/simulator side ):

Reply:
That's unfortunate, thanks for clarifying!